tests/e2e/actions.go

```diff
@@ -1844,3 +1844,25 @@ func (tr TestRun) GetPathNameForGorelayer(chainA, chainB chainID) string {
 
 	return pathName
 }
+
```

Comment:
The last Hermes version requires now to run the evidence cmd to detect consumer misbehaviours.